DSharpPlus/Enums/InteractionType.cs

```diff
@@ -29,6 +29,7 @@ namespace DSharpPlus
     public enum InteractionType
     {
         Ping = 1,
-        ApplicationCommand
+        ApplicationCommand,
+        Component = 3
```

Comment:
The order is 1, 2, 3. 3 doesn't need to have a value specified. Or, why not specify a value for 2 as well?